lib/logflare_web/live/search_live/logs_search_lv.ex

```diff
@@ -8,6 +8,7 @@ defmodule LogflareWeb.Source.SearchLV do
   import LogflareWeb.ModalLiveHelpers
   import LogflareWeb.SearchLV.Utils
   import LogflareWeb.SearchLive.TimezoneComponent
+  import LogflareWeb.QueryComponents, only: [humanize_bytes: 1]
```

Comment:
Utils module alias would be better, imports make global find/replace refactoring more tricky